parallel-install/pkg/components/prerequisites.go

```diff
@@ -38,7 +40,7 @@ func NewPrerequisitesProvider(overridesProvider overrides.OverridesProvider, res
 		resourcesPath:     resourcesPath,
 		components:        components,
 		helmConfig:        helmCfg,
-		log:               cfg.Log,
+		log:               logger.NewLogger(cfg.Verbose),
```

Comment:
Previous version was actually shorter and simpler. What's the benefit here?

Reply:
It was discussed to create a separate logger for every component. After followup, we have decided to change the strategy.